### ISSUE TITLE
Seems to work no longer with latest chrome / firefox

### ISSUE THREAD

**Issue by fyyyyy** [borismus/MagicTouch#12]

**Comment by tory** [comment]
Hey @fyyyyy - did you find another library/plugin/etc to accomplish what you're after? I'm seeing the same as you.

**Comment by fyyyyy** [comment]
sorry, havent found anything else than working with an ipad in dev mode

**Comment by maboiteaspam** [comment]
yeah, it s terrible, for some reasons http://borismus.github.io/mobile-web-samples/browser-ninja/ throws a fatal here on chrome.

Uncaught TypeError: Failed to execute 'fillRect' on 'CanvasRenderingContext2D': 4 arguments required, but only 0 present
